metrics/sinks/stackdriver/stackdriver.go

```diff
@@ -613,6 +613,9 @@ func (sink *StackdriverSink) TranslateMetric(timestamp time.Time, labels map[str
 				"memory_type": "evictable",
 			}
 			return ts
+		case "nvidia.com/gpu/request":
```

Comment:
Keep in mind that this exposes the metrics for customers using new resource model. New resource model is going to be opt-in in GKE for Kubernetes 1.10 and legacy resource modes is still going to be a default. If you want to expose this metric also to customers with new resource model, please add a second (legacy metric). The appropriate name would be "container.googleapis.com/container/nvidia_gpu/request" and it should be reported against legacy "gke_container" resource type.